pkg/cli-runtime/printer/resource.go

```diff
@@ -91,6 +91,22 @@ func OutputResource(obj Object, format OutputFormat, scheme *runtime.Scheme) (st
 	return printObject(copy, format)
 }
 
+func OutputResources(objList []Object, format OutputFormat, scheme *runtime.Scheme) (string, error) {
```

Comment:
Can these two functions be consolidated `OutputResource` & `OutputResources`?

Reply:
+1